src/components/context-menu/ContextMenu.svelte

```diff
@@ -18,27 +33,37 @@
     y = e.clientY;
   }
 
+  export function getBoundingClientRect() {
```

Comment:
are you using this function? i wasn't able to see any references to it

Reply:
Ah good catch, not used any longer, thanks!